src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrush_Stretch.cs

```diff
@@ -1,55 +0,0 @@
-﻿using System.Drawing;
-using System.Linq;
-using NUnit.Framework;
-using SamplesApp.UITests.TestFramework;
-using Uno.UITest.Helpers.Queries;
-
-namespace SamplesApp.UITests.Windows_UI_Xaml_Media.ImageBrushTests
-{
-	[TestFixture]
-	public partial class ImageBrush_Stretch : SampleControlUITestBase
-	{
-		private const string Redish = "#FFEB1C24";
-		private const string Yellowish = "#FFFEF200";
-		private const string Greenish = "#FF0ED145";
-
-		[Test]
-		[AutoRetry]
-
-		[ActivePlatforms(Platform.Browser, Platform.iOS)] // Android behaves differently. See https://github.com/unoplatform/uno/pull/7238#issuecomment-937667565
-		public void When_Stretch()
-		{
-			Run("UITests.Windows_UI_Xaml_Media.ImageBrushTests.ImageBrush_Stretch");
```

Comment:
I'm not sure if we should remove the sample from SamplesApp.

@jeromelaban What do you think?

Reply:
According to the issue the goal was to "decrease significantly the CI build time". I may be counterproductive to keep both, no?

Reply:
@amelield I mean https://github.com/amelield/uno/blob/dev/ALD/test-move/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrush_Stretch.xaml and https://github.com/amelield/uno/blob/dev/ALD/test-move/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrush_Stretch.xaml.cs which still exist.

But please wait for @jeromelaban opinion first before deleting them.

Reply:
After seeing the CI result I will remove MacOs from the test. The When_Opacity test now fail I have not changed the value, I think I just tested with When_Inner_Opacity() , my bad. Do I change the expected value for the one the CI wants #FFFF0000, when #FFE6E6E6 it is written? The runtime test is giving me the same result. I also would wait to see if I delete the files or not and just make a commit with all the fixes.

Reply:
I don't think you need to change `When_Opacity` test. Your changes shouldn't be affecting it and it's likely an intermittent failure.

Reply:
Indeed, this file cannot be removed at this time. In this PR, we can now mark the test with `[ActivePlatforms(Platform.Browser)]` so it runs only on wasm. We'll delete this file once we're able to use `WriteableBitmap` on Wasm.

Reply:
[ActivePlatforms(Platform.Browser)] does not work in runtimes and it was already in a #if __SKIA__ section (that cover all the test of that class) could I just comment that single test section for the moment being(with an actual comment)? All other subtest are passing.

Reply:
@amelield I think what @jeromelaban meant is to have the test both as UI test and runtime test.

- UI test: Runs only on Wasm.
- Runtime test: Runs on other platforms.

@jeromelaban Could you confirm?

Reply:
That make sense for the sample app right. I mixed up the two problems.

Reply:
> I don't think you need to change `When_Opacity` test. Your changes shouldn't be affecting it and it's likely an intermittent failure.

It's not an intermittent failure, I was wrong.

Though I don't clearly see why the test broke.